bin/zipignore

```diff
@@ -0,0 +1,24 @@
+.git
+
+shared/platform/content/
+bin/
+shim/
+
+./lint
+./run
+./test
+
+test/
+tests/
+
+docs/
+
+.*
+*.md
+*.sh
+react-dev.js
```

Comment:
zipignore is for building the xpi.
When you are using ./run or ./test, the xpi isn't built.
Instead, firefox refers directly to the devtools add-on folder. i.e. repo's root folder.
So that all files are available.